e2e/pkg/auth/auth.go

```diff
@@ -24,8 +25,14 @@ func BuildJwtForUser(ctx context.Context, user string) (string, error) {
 	}
 
 	block, _ := pem.Decode(s.Data["private"])
-	parseResult, _ := x509.ParsePKCS8PrivateKey(block.Bytes)
-	key := parseResult.(*rsa.PrivateKey)
+	parseResult, err := x509.ParsePKCS8PrivateKey(block.Bytes)
+	if err != nil {
+		return "", err
+	}
+	key, ok := parseResult.(*rsa.PrivateKey)
+	if !ok {
+		return "", fmt.Errorf("Failed to retrieve RSA private key")
```

Comment:
IIRC, as of now we are using lowercase for the first letter of the error message

```suggestion
		return "", fmt.Errorf("failed to retrieve RSA private key")
```